src/core/control/jobs/RenderJob.cpp

```diff
@@ -93,15 +93,15 @@ static void repaintWidgetArea(GtkWidget* widget, int x1, int y1, int x2, int y2)
     Util::execInUiThread([=]() { gtk_xournal_repaint_area(widget, x1, y1, x2, y2); });
 }
 
-void RenderJob::repaintPage() const {
-    repaintPageArea(0, 0, view->getWidth(), view->getHeight());
-}
+void RenderJob::repaintPage() const { repaintPageArea(0, 0, view->getWidth(), view->getHeight()); }
 
 void RenderJob::repaintPageArea(double x1, double y1, double x2, double y2) const {
     double zoom = view->xournal->getZoom();
     int x = view->getX();
     int y = view->getY();
-    repaintWidgetArea(view->xournal->getWidget(), x + std::floor(zoom * x1), y + std::floor(zoom * y1), x + std::ceil(zoom * x2), y + std::ceil(zoom * y2));
+    repaintWidgetArea(view->xournal->getWidget(), x + static_cast<int>(std::floor(zoom * x1)),
```

Comment:
Here you could use my floor_cast.

Reply:
In [6a29634](https://github.com/xournalpp/xournalpp/pull/5328/commits/6a2963414ac8d3dac053e9d9029ac1506a9f0deb) I replaced `static_cast<int>(std::floor)` by `floor_cast` everywhere, and similarly with `ceil_cast` and `round_cast`. It improves readability.